worlds/peaks_of_yore/items.py

```diff
@@ -0,0 +1,6 @@
+from BaseClasses import Item
+
+
+class PeaksOfYoreItem(Item):
+    game = "Peaks of Yore"
+
```

Comment:
Since you're only using this class in one spot in your `__init__.py`, you can probably just move the class directly to your `__init__.py` and remove this file. Or move some more stuff to here. Mainly it feels a little weird that it's only housing this one class.